packages/v4/src/content/design-guidelines/components/form/forms.md

```diff
@@ -22,273 +22,375 @@ related: [
 * First name, Middle initial, Last name
-* City, State, Zip code
+* City, State, ZIP code
 * Credit card number, Expiration date, Security code
 
-<img src="./img/multi-column-eg.png" alt="Example of correct and incorrect multi-column forms" />
+<img src="./img/multi-column-eg.png" alt="Example of correct and incorrect multi-column forms"/>
+
+Never design multi-column forms to read like a book, where the left column is completed first before moving on to the right. The form itself should still move and be filled from top to bottom, with related fields placed on the same line.
+
+<img src="./img/multi-column-read.png" alt="Example of how multi-column forms should be read and presented, demonstrated by arrows that follow a user's path of consumption"/>
+
+## Content
+
+When writing form content, use your understanding of the user’s task and context to plan your content hierarchy. As a user moves through a form, they input and select information based on the content provided, almost like a conversation. 
+
+Forms combine several content types to guide users toward completion:
+
+* Field labels
+* Placeholder text
+* Helper text
+* Popovers
+* Error messages
+* Buttons
+
+Follow these general guidelines for each content type as you structure content within a form.
+
+### Field labels
+
+Include the most critical information about what data the user needs to provide. Each label should be unique and identify all the information being collected. 
+
+Whenever possible, keep field labels under 5 words and avoid using the same field label more than once in the same form.
+
+Examples:
+
+* *Full name*
+* *Email*
+* *Phone number*
+
+### Placeholder text
+
+Include examples or formatting expectations to inform users about specifications needed to input information without errors. 
+
+If your placeholder text shares an example of a correct input, introduce it as an example so that users can distinguish impermanent examples from field labels and their own inputted text.
+
+Examples:
+* *Example, John Smith*
+* *Example, (555) 555-5555*
+
+Placeholder text that doesn’t give an example doesn’t need to be introduced as one. If your placeholder text simply communicates a field's required format, write the format without an introduction.
+
+Examples:
+* *MM-DD-YYYY*
+* *johnsmith@example.com*
+
+**Note:** Once users start typing, placeholder text disappears. Consider communicating crucial formatting requirements through helper text instead.
+
+
+
+See our [text input guidelines](/components/text-input/design-guidelines) for more specific guidance on writing placeholder text for text input fields.
+
+### Helper text
+
+Include critical secondary information, the information a user must have in order to provide the right input. Let the user know any conditions for their inputted information to avoid any unnecessary errors, like specifying that an inputted name for a new asset must be a unique one.
```

Comment:
I feel like I would "as helper text" after "include critical secondary information". This whole first sentence reads a little weird to me?

Reply:
Hm, yeah, let's get ride of that second clause and rework it. How does this sound?:

"Include critical secondary information to help users input correct information on their first attempt."